.env

```
@@ -60,8 +53,17 @@ ENABLE_ASSET_CACHING=true
 # files on disk are considered fresh rather than reacqiring them. In hours.
 ASSET_CACHING_PERIOD=72
 
-# Just samples, you need to set these (remember: JSON formatted)
-BACKEND_CORS_ORIGINS='["http://localhost", "http://localhost:8080"]'
+#  * http://localhost:3000 covers requests originating from the case
+#  where 'npm run dev' is invoked to run vite outside Docker. This
+#  results in vite's development mode which runs on port 3000.
+#  * http://localhost:8001 covers requests originating from the case
+#  where 'npm run build' is invoked which executes a vite build. Docker
+#  then copies built assets to nginx environment which then serves them
+#  on port 80 which is then mapped to port 8001 in docker-compose.yml.
+#  * https://irguidev.walink.org covers requests originating from the
+#  development/staging server for the (G)UI at WA.
+#  (Array must be JSON formatted)
+BACKEND_CORS_ORIGINS='["http://localhost:3000", "http://localhost:8001", "https://irguidev.walink.org"]'
```

```suggestion
BACKEND_CORS_ORIGINS='["http://localhost:3000", "http://localhost:8001"]'
```
we can leave it like this and set the dev/prod CORS origins via environment variable. I just did this as a hack.

@linearcombination can you make this change so I can merge the PR?

---

This is complete. Thank you.